### ISSUE TITLE
Initial position wrong when used with details / summary element

### ISSUE THREAD

**Issue by AndreasHald** [popperjs/popper-core#1339]
Hi, I think I discovered a bug while working with popper and the details / summary element. When using popper on the content of the details element, the initially calculated position appears wrong, when toggling the element. any resizing of the window will correct the position and it will be correct until page reload. I've managed to circumvent the issue by adding a call to ```instance.forceUpdate()``` when toggling the details element. But I thought I would report the issue anyway.

### CodeSandbox demo

[reproducible example](https://codesandbox.io/s/popperjs-bug-repro-rh4uv)

### Steps to reproduce the problem

1. Click on the summary element and notice the content is out of view
![Screenshot 2021-08-08 at 19 12 11](https://user-images.githubusercontent.com/8914722/128639967-acb8a1f0-5710-43dc-8c73-74d0f1fa8b64.png)

2. Resizing the window or causing the instance to update after having opened the details element cause the correct position
![Screenshot 2021-08-08 at 19 12 45](https://user-images.githubusercontent.com/8914722/128639978-65f96041-d976-4d39-99a3-aeec98598629.png)

### What is the expected behavior?
I would expect the initial position to be correct.

### What went wrong?
I assume it has to do with the way the details element hides the content until toggled open

### Any other comments?
[docs on details element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details)
[docs on summary element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary)

**Comment by FezVrasta** [comment]
I don't think this is a bug, in the docs we specify the element must be attached to the DOM and have a position in order to work.

I think your solution is the best way to address this case.

**Comment by AndreasHald** [comment]
You may be right that it's unfixable. But to my knowledge the element is attatched to the dom, i'm unsure about it's position while hidden, where in the docs do you refer to? And which browser api's do you use?

I can investigate it abit.
